assets/images/withdraw-methods/WireCard.svg

```diff
@@ -0,0 +1,10 @@
+</defs>
+</svg>
```

Comment:
Kindly change the file name format for all of these assets according to the existing convention. for example, `wirecard.svg`, `stripe.svg`, `razor-pay.svg`